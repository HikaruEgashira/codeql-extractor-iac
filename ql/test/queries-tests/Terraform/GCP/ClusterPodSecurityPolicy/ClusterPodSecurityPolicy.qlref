security/Terraform/GCP/ClusterPodSecurityPolicy.ql