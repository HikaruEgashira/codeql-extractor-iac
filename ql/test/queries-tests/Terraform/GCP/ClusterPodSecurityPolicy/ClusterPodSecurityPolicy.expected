| pods.hcl:7:1:15:1 | resource google_container_cluster cluster2 | No Pod Security Policy Defined |
